src/DoctrineMongoODMModule/Service/ConfigurationFactory.php

```diff
@@ -98,6 +98,11 @@ public function __invoke(ContainerInterface $container, $requestedName, ?array $
             }
         }
 
+        $className = $options->getDefaultDocumentRepositoryClassName();
+        if ($className) {
+            $config->setDefaultDocumentRepositoryClassName($className);
```

Comment:
This method is available since ODM 1.3 while module allows `^1.1`. You'll need to either bump ODM or add an if with `method_exists` falling back to `setDefaultRepositoryClassName`

Reply:
Fixed with https://github.com/doctrine/DoctrineMongoODMModule/pull/220/commits/db6bbb37132d82337e055557aaa0f53a23fe9aa6